packages/osc-ui/src/components/Button/button.scss

```
@@ -187,6 +187,21 @@
                 )
             )
         ),
+        septenary: (
+            bg-color: transparent,
+            bg-hover-color: var(--btn-hover-color-tertiary),
+            fg-color: var(--color-tertiary),
+            fg-hover-color: var(--btn-fg-hover-color-tertiary),
+            font-weight: $fw-reg,
+            text-transform: capitalize,
+            border-bottom: 1px solid var(--color-tertiary),
+            sizes: (
+                lg: (
+                    padding: $space-l $space-3xl,
```

I would reduce the size of the vertical padding to `$space-m`. It'll bring the height of the buttons down to closer match the design. They feel a bit big at the moment.
```suggestion
                    padding: $space-m $space-3xl,
```